packages/grid/_modules_/grid/components/editCell/EditInputCell.tsx

```diff
@@ -39,15 +41,11 @@ export function EditInputCell(props: GridCellParams & InputBaseProps) {
 
   const onKeyDown = React.useCallback(
     (event: React.KeyboardEvent) => {
-      if (!inputBaseProps.error && event.key === 'Enter') {
+      if (!inputBaseProps.error && (event.key === 'Enter' || event.key === 'Tab')) {
```

Comment:
It would be great to not have to map the onKeyDown event and just subscribe to on commit changes.
So the commit trigger (enter/tab) is abstracted of the component.
something like

```tsx
const onChangeCommitted = React.useCallback((inputProps, event)=> {
  if (!inputProps.error) {
     event.preventDefault();
     return false; // something to stop the event from continuing,
  }
})
```

ATM we don't provide a way to do block the event, like `preventDefault`;